### PR TITLE
5 Day Weather Forecase API

### PR DESCRIPTION
**Pull request by chanacham**

### CODE REVIEW

File: src/main/java/weather/Coord.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+package weather;
+
+public class Coord {
+
+    private Double mLat;
```
Comment (andrewoid) [inline src/main/java/weather/Coord.java]: I'm guessing this doesn't actually work.

File: src/test/java/chambre/weather/CurrentWeatherTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                .addConverterFactory(GsonConverterFactory.create())
+                .addCallAdapterFactory(RxJava3CallAdapterFactory.create())
+                .build();
+        WeatherService service = retrofit.create(WeatherService.class);
+
+        //when
+       CurrentWeather weather = service.getCurrentWeather("New York").blockingFirst();
+
+        //then
+        assertNotNull(weather);
```
Comment (andrewoid) [inline src/test/java/chambre/weather/CurrentWeatherTest.java:29]: The result is not null, but `weather` probably doesn't have any data.

**Review by andrewoid — CHANGES_REQUESTED** [review]

File: src/test/java/chambre/weather/CurrentWeatherTest.java (hunk 2 above)
Comment (chanacham) [reply src/test/java/chambre/weather/CurrentWeatherTest.java:29]: What does that mean?

File: src/main/java/weather/currentClasses/CurrentWeather.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
+
+package weather.currentClasses;
```
Comment (andrewoid) [inline src/main/java/weather/currentClasses/CurrentWeather.java:2]: just `current`

package names should not have capital letters.

File: src/main/java/weather/forecastClasses/City.java [hunk 4]
```diff
@@ -0,0 +1,15 @@
+
+package weather.forecastClasses;
```
Comment (andrewoid) [inline src/main/java/weather/forecastClasses/City.java:2]: just `forecast`

File: src/test/java/chambre/weather/CurrentWeatherTest.java (hunk 2 above)
Comment (andrewoid) [reply src/test/java/chambre/weather/CurrentWeatherTest.java:29]: Can you add an assert here that data inside of weather is not null?

File: src/test/java/chambre/weather/CurrentWeatherTest.java [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+                .addCallAdapterFactory(RxJava3CallAdapterFactory.create())
+                .build();
+        WeatherService service = retrofit.create(WeatherService.class);
+
+        //when
+        CurrentWeather weather = service.getCurrentWeather("New York").blockingFirst();
+
+        //then
+        assertNotNull(weather);
+        assertNotNull(service.getCurrentWeather("New York"));
```
Comment (andrewoid) [inline src/test/java/chambre/weather/CurrentWeatherTest.java:30]: No. I asked you to check the data INSIDE `weather`.

File: src/main/java/weather/chambre/Clouds.java [hunk 6]
```diff
@@ -0,0 +1,9 @@
+
+package weather.chambre;
```
Comment (andrewoid) [inline src/main/java/weather/chambre/Clouds.java:2]: no idea why you added a `chambre` package and only put some classes inside the package.

**Pull request closed without merging**